ingestion/src/metadata/ingestion/source/database/dynamodb.py

```diff
@@ -46,6 +46,15 @@ def __init__(self, config: WorkflowSource, metadata_config: OpenMetadataConnecti
         )
         self.metadata_config = metadata_config
         self.metadata = OpenMetadata(metadata_config)
+        service = self.metadata.get_by_name(
+            entity=DatabaseService, fqn=self.config.serviceName
+        )
+        if service:
+            self.config.serviceConnection = (
+                self.metadata.secrets_manager_client.retrieve_service_connection(
+                    service, "database"
```

Comment:
these `database`, `dashboard`... Maybe we can just prepare an Enum somewhere and use the values from there

Reply:
Added `ServiceType` enum.